src/audio/volume.c

```diff
@@ -143,23 +143,6 @@ static uint64_t vol_work(void *data)
 		return 0;
 }
 
-/**
- * \brief Validates and sets minimum and maximum volume levels.
- * \details If max_vol < min_vol or it's equals 0 then set max_vol = VOL_MAX
- * \param[in,out] cd Volume component private data.
- * \param[in] min_vol Minimum volume level
- * \param[in] max_vol Maximum volume level
- */
-static void vol_set_min_max_levels(struct comp_data *cd,
```

Comment:
Actually I realized that this kind of function is needed. The volume request from IPC need to be limited to be within VOL_MIN .. VOL_MAX.

Reply:
The check is there in volume_set_chan() so it's OK already.